src/pages/MetricDetail.svelte

```diff
@@ -204,6 +207,14 @@
       <td>Version</td>
       <td>{metric.version || 0}</td>
     </tr>
+    {#if searchfoxMap[params.app]}
+      <tr>
+        <td>Find in</td>
```

Comment:
Let's just use "Searchfox" here. It's on me to add some help text on what this means. 😄 

```suggestion
        <td>Searchfox</td>
```